engine/access/rpc/backend/connection_factory_test.go

```diff
@@ -88,7 +79,7 @@ func TestProxyExecutionAPI(t *testing.T) {
 	}
 
 	// get an execution API client
-	client, err := proxyConnectionFactory.GetExecutionAPIClient("foo")
+	client, _, err := proxyConnectionFactory.GetExecutionAPIClient("foo")
```

Comment:
please close the noop closer in at least one of these tests to ensure it works as expected